client/Makefile

```diff
@@ -37,11 +37,11 @@ start-frontend:
 .PHONY: smoke-test
 smoke-test:
 	start_server_and_test \
-		'../launch_dev_server.sh --config-file $(CXG_CONFIG) --port $(CXG_SERVER_PORT)' \
+		'CELLXGENE_DISABLE_AUTH=true ../launch_dev_server.sh --config-file $(CXG_CONFIG) --port $(CXG_SERVER_PORT)' \
 		$(CXG_SERVER_PORT) \
 		$(if $(and $(CURRENT_SHARD),$(TOTAL_SHARDS)), \
-			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --shard=$(CURRENT_SHARD)/$(TOTAL_SHARDS)', \
-			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e')
+			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --project=chromium --shard=$(CURRENT_SHARD)/$(TOTAL_SHARDS)', \
```

Comment:
Client make target switches to `playwright test --project=chromium` so we can add the second project without running everything twice.